sdk/spring/azure-spring-integration-servicebus/src/main/java/com/azure/spring/integration/servicebus/converter/ServiceBusMessageConverter.java

```diff
@@ -24,14 +25,28 @@
 import java.util.UUID;
 
 /**
- * A converter to turn a {@link org.springframework.messaging.Message} to {@link IMessage}
- * and vice versa.
+ * A converter to turn a {@link org.springframework.messaging.Message} to {@link IMessage} and vice versa.
  *
  * @author Warren Zhu
  */
 public class ServiceBusMessageConverter extends AbstractAzureMessageConverter<IMessage> {
     private static final Logger LOG = LoggerFactory.getLogger(ServiceBusMessageConverter.class);
 
+    private final ObjectMapper objectMapper;
+
+    public ServiceBusMessageConverter() {
+        objectMapper = new ObjectMapper();
```

Comment:
Can we use the static field in parent class?
```
objectMapper = OBJECT_MAPPER;
```
Change `OBJECT_MAPPER` to `protected`.